python/tests/test_pyvw.py

```diff
@@ -353,6 +353,58 @@ def test_example_features():
     assert ex.pop_namespace()
 
 
+def count_weights_from_readable_model_file(file_name):
```

Comment:
I also like the use of python for testing, good precedence. BTW, I recently added a new python test file called test_cb so it might be worth migrating this to over there? your call

Reply:
this and the other test

Reply:
Yeah easily done, moving to a test_ccb file